### ISSUE TITLE
Incorrectly raises X11Error('Malformed symbol string')

### ISSUE THREAD

**Issue by eddie-dunn** [hcpl/xkbgroup#3]
```
$ setxkbmap -query
rules:      evdev
model:      pc105
layout:     us(altgr-intl),se
options:    caps:escape,grp:shifts_toggle,grp_led:caps
```

With xkbgroup:
```
In [13]: import xkbgroup
In [14]: xkb = xkbgroup.XKeyboard()

In [15]: xkb.groups_symbols
---------------------------------------------------------------------------
X11Error                                  Traceback (most recent call last)
<ipython-input-15-af8c17d27d99> in <module>()
----> 1 xkb.groups_symbols

test/lib/python3.5/site-packages/xkbgroup/core.py in groups_symbols(self)
    269         :type: list of str
    270         """
--> 271         return _ListProxy(symdata.symbol for symdata in self._symboldata_list)
    272 
    273     @property

test/lib/python3.5/site-packages/xkbgroup/core.py in _symboldata_list(self)
    436         if symbol_str_atom != None_:
    437             b_symbol_str = XGetAtomName(self._display, symbol_str_atom)
--> 438             return _parse_symbols(b_symbol_str.decode(), self.non_symbols)
    439         else:
    440             raise X11Error("Failed to get symbol names.")

test/lib/python3.5/site-packages/xkbgroup/core.py in _parse_symbols(symbols_str, non_symbols, default_index)
    489     symboldata_list = []
    490     for symstr in symbols_str.split('+'):
--> 491         symboldata = get_symboldata(symstr)
    492         if symboldata.symbol not in non_symbols:
    493             symboldata_list.append(symboldata)

test/lib/python3.5/site-packages/xkbgroup/core.py in get_symboldata(symstr)
    485                 int(index) - 1 if index else default_index)
    486         else:
--> 487             raise X11Error("Malformed symbol string: \"{}\"".format(symstr))
    488 
    489     symboldata_list = []

X11Error: Malformed symbol string: "us(altgr-intl)"
```

**Comment by eddie-dunn** [comment]
solved by #5

**Comment by eddie-dunn** [comment]
@hcpl could you please take a look at my two pull requests?

**Comment by eddie-dunn** [comment]
Fix merged in 2511044ad25c15b85d430c5331e8e2266e2907be
